pom.xml

```diff
@@ -318,5 +318,22 @@
         </pluginManagement>
       </build>
     </profile>
+    <profile>
+      <id>travis</id>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-compiler-plugin</artifactId>
+            <version>3.6.2</version>
+            <configuration>
+              <verbose>true</verbose>
+              <fork>true</fork>
+              <executable>/usr/lib/jvm/java-8-oracle/bin/javac</executable>
```

Comment:
java.util.Base64 is not available before Java 8 so it can't be built on SDK 6 and 7. It can be built on Java 8 and executed on 6 and 7. if you really need it to be built on SDK 6 and 7, feel free to close this PR.